### PR TITLE
acrn-config: Update generated configuration source code

### PR DESCRIPTION
**Pull request by NanlinXie**
1. Reorg generated configuration source code structure
2. Upstream generated configuration source code based on generic board infomation
3. Update license date from 2020 to 2021

Tracked-On: #5644
Signed-off-by: Xie, nanlin <nanlin.xie@intel.com>

### CODE REVIEW

File: misc/config_tools/data/generic_board/generic_code/hybrid/pci_dev.d [hunk 1]
```diff
@@ -0,0 +1,27 @@
+/home/nlxie/samba/branches/github/nlxie_github/acrn-hypervisor-1/build/hypervisor/configs/scenarios/hybrid/pci_dev.o: \
```
Comment (gvancuts) [inline misc/config_tools/data/generic_board/generic_code/hybrid/pci_dev.d:1]: It looks like this file shouldn't be here.

File: misc/config_tools/data/generic_board/generic_code/hybrid/vm_configurations.d [hunk 2]
```diff
@@ -0,0 +1,22 @@
+/home/nlxie/samba/branches/github/nlxie_github/acrn-hypervisor-1/build/hypervisor/configs/scenarios/hybrid/vm_configurations.o: \
```
Comment (gvancuts) [inline misc/config_tools/data/generic_board/generic_code/hybrid/vm_configurations.d:1]: Please remove this file.

**Review by gvancuts — CHANGES_REQUESTED** [review]
There are many files with copyright information indicating year 2020 (or older). These should be updated to 2021.

You have a number of `*.d` and object (`*.o`) files included in these commits (e.g. `misc/config_tools/data/generic_board/generic_code/hybrid/pci_dev.o`), can you remove them?

**Comment by HuiHuangShi** [comment]
No new violations to the coding guideline detected.<br>No New Name Conflict<br>No New Function Declaration/Definition Mismatch

**Comment by wenlingz** [comment]
@NanlinXie  please rebase your code, thanks

File: misc/config_tools/data/generic_board/generic_code/hybrid/vm_configurations.d (hunk 2 above)
Comment (NanlinXie) [reply misc/config_tools/data/generic_board/generic_code/hybrid/vm_configurations.d]: Done

**Review by terryzouhao — APPROVED** [review]

**Pull request merged**